data/nsides/offsides/meta.yaml

```diff
@@ -0,0 +1,39 @@
+---
+name: offsides
+description: OffSIDES is a database of individual drug side effect signals mined from the FDA's Adverse Event Reporting System. The innovation of OffSIDES
+    is that a propensity score matching (PSM) model is used to identify control drugs and produce better PRR estimates. In OffSIDES we focus on drug safety
+    signals that are not already established by being listed on the structured product label -- hence they are off-label drug side effects.
+targets:
+    - id: PRR
+      description: Proportional reporting ratio)
+      type: continuous
+      names:
+          - Proportional reporting ratio
+    - id: PRR_error
+      description: Standard error of the PRR estimate
+      type: continuous
+      names:
+          - Proportional reporting ratio error
+    - id: mean_reporting_frequency
+      description: Proportion of reports for the drug that report the side effect
+      type: continuous
+      names:
+          - mean reporting frequency
+identifier:
+    - id: drug_concept_name
+      description: RxNorm name string for the drug
+      type: categorical
+    - id: condition_concept_name
+      description: MedDRA identifier for the side effect
+      type: categorical
+license: CC BY 4.0
+links:
+    - url: https://tatonettilab.org/resources/nsides/
+      description: data source
+    - url: https://nsides.io/
+      description: database website
+num_points: 3042873
+bibtex: "\n        @article{Tatonetti2012,\n        author = {Tatonetti, Nicholas P. and Ye, Peter P. and Daneshjou, Roxana and Altman, Russ B.},\n    \
+    \    title = {Data-driven prediction of drug effects and interactions},\n        journal = {Sci Transl Med},\n        volume = {4},\n        number\
+    \ = {125},\n        pages = {125ra31},\n        year = {2012},\n        doi = {10.1126/scitranslmed.3003377},\n        pmid = {22422992},\n        pmcid\
+    \ = {PMC3382018}\n        }\n        "
```

Comment:
can we remove those newlines somehow and just have a multiline string? I can help with that

Reply:
sure! i'll let you take care of it if thats ok